packages/create-cloudflare/templates/next/env.d.ts

```diff
@@ -1,7 +1,5 @@
+// Generated by Wrangler
```

Comment:
Was the lack of this causing an initial build failure 😄 ?

Reply:
it was causing the `build-cf-types` script to fail since that would refuse to override the file 🙂 

https://github.com/cloudflare/workers-sdk/blob/1720f0a12a6376093b3c5799d74f47c522ae8571/packages/wrangler/src/type-generation.ts#L331-L340